src/protocolsupportpocketstuff/zplatform/PlatformThings.java

```diff
@@ -17,9 +16,6 @@
 
 	public static void bakeStuff() {
 		switch(ServerPlatformIdentifier.get()) {
-		case GLOWSTONE:
-			stuff = new GlowstoneStuff();
-			break;
```

Comment:
Do we still want to support Glowstone, long term? Is anyone using or testing on Glowstone? (Asking just as much about PSPE as PSPS).